docs/ica-tally-tutorial.md

```
@@ -0,0 +1,404 @@
+cd typescript/cli
+pnpm hyperlane ica deploy \\
+  --origin ethereum \\
+  --chains optimism \\
+  --owner <YOUR_DAO_GOVERNOR_ADDRESS>
+\\\
+
+The CLI will output:
+- Whether an ICA already exists (showing the address)
+- Or deploy a new one (showing the deployed address)
+
+### Using the SDK Programmatically
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)